packages/expo-dev-launcher/ios/DevLauncherAuth.swift

```diff
@@ -0,0 +1,85 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+import SafariServices
+import React
+
+private let DEV_LAUNCHER_DEFAULT_SCHEME = "expo-dev-launcher"
+
+public class DevLauncherAuth: Module {
+  private var redirectPromise: Promise?
+  private var authSession: SFAuthenticationSession?
+
+  public func definition() -> ModuleDefinition {
+    Name("ExpoDevLauncherAuth")
+
+    AsyncFunction("openAuthSessionAsync") { (authURL: String, redirectURL: String, promise: Promise) in
+      try self.initializeWebBrowser(promise)
+
+      if #available(iOS 11.0, *) {
+        guard let url = URL(string: authURL) else {
```

Comment:
I think you can just use `URL` instead of `String` for the `authURL` argument, then this guard will be unnecessary

Reply:
updated